### DISCUSSION TITLE
f0028 squash merged into develop

### DISCUSSION THREAD

**Discussion by adesutherland** [adesutherland/CREXX#80] [Progress Updates]
As discussed - Now we can merge from develop - I will continue with function calling in a new branch

**Comment by Peter-Jacob** [comment]
Please advise, what must I do (I am a novice in using GIT). Updating my f0028 says "can't update feature/f0028 has no branch". Should I switch to development and update from there?

**Comment by adesutherland** [comment]
Yes switch to develop ... but I suggest you manually save changed files
first ...

Does anyone have good instructions for Peter to do a pull request?

Apologies for difficulties!

**Reply by adesutherland** [reply]
Suggestion 2: Rene / Peter you could share a branch as you are working closely together. I will suggest a number - hold the line!

**Reply by Peter-Jacob** [reply]
That makes sense!

**Reply by adesutherland** [reply]
It will be branch feature/f0030. I will create it now. A

**Reply by Peter-Jacob** [reply]
Did you add f0030? I can't find it.

**Reply by adesutherland** [reply]
Morning. Yes I did: feature/f0030.

You may need to do a git fetch to update you local repository. Then you should be able to check it out. I think ...

**Comment by rvjansen** [comment]
I checked in the rxas rexx functions length, copies, substr, right and left. Please have at it. When testing on ubuntu aarch64 I noticed some things with spurious characters, subsequently mostly fixed, only left() still shows one. Probably my bugs and not in the 'microcode' but you never know.

**Reply by rvjansen** [reply]
and they are in lib/rxfns. Also did a small change in machine/CMakelists.txt to enable other rexxen than Regina
